src/components/Layout/Header/ActionCenter/components/AppUpdateActionCard.tsx

```diff
@@ -6,7 +6,7 @@ import { useTranslate } from 'react-polyglot'
 
 import { ActionCard } from './ActionCard'
 
-import UpdateIcon from '@/assets/update-icon.svg'
+import UpdateIcon from '@/assets/update-icon.svg?url'
```

Comment:
Absolutely unrelated but I have to cherry-pick every single time I run the dev build or I can't run it so...

Would be nice to find a systematic way to fix those without having to go back and forth tbh.